content/en/Platform Deep Dive/Cobalt Account/account-settings.md

```diff
@@ -0,0 +1,118 @@
+
+### Change Your Password
+
+Follow this instruction if you remember your password and want to change it. {{% forgot-password-reset %}}
+
+1. Navigate to https://app.cobalt.io/settings/account.
+1. In the **New password** field, enter a new password.
+1. Enter the same password in the **Password Confirmation** field.
+1. Enter your current password to confirm the changes, and select **Update Account**.
+
+## Manage Security Settings
+
+To manage two-factor authentication (2FA) settings for your account, select **Security Settings** on the user menu.
+
+### Enable Two-Factor Authentication
+
+Even if your organization hasn't enforced 2FA for all users, we recommend that you enable it to enhance your account security. If you're using [SAML SSO](/getting-started/sign-in/#saml-sso) to sign in, you don't need to turn on 2FA.
+
+![Configure 2FA for your account](/deepdive/SecuritySettings.png "Two-factor authentication page")
+
+To enable 2FA on your account:
+
+1. Navigate to https://app.cobalt.io/settings/security/2fa.
+    - You should see **Disabled** under **Status**.
+1. Enter your current password, and select **Enable two-factor authentication**.
+    - {{% forgot-password-reset %}}
+1. Scan the QR code using an authenticator app on your mobile device, such as [Authy](https://authy.com/) or [Google Authenticator](https://support.google.com/accounts/answer/1066447).
+1. Enter a 6-character code that your authenticator app generates.
+1. Select **Enable two-factor authentication** to confirm.
+
+Now, each time you sign in to Cobalt, you need to enter a one-time code from your authenticator app. You can turn on the option to remember your device for 30 days.
+
+For troubleshooting tips, see [Problems with Two-Factor Authentication](/platform-deep-dive/cobalt-account/account-recovery/#problems-with-two-factor-authentication).
+
+### Reset Two-Factor Authentication
+
+We don't recommend turning off 2FA on your account. However, you may need to reset 2FA when:
+
+- You want to switch to a different authenticator provider.
+- You believe your [account was compromised](/platform-deep-dive/cobalt-account/account-recovery/#account-was-compromised)—and you still have access to your authenticator.
+- You have a new device. Depending on your authenticator, you can transfer data to a new device without having to reset 2FA in Cobalt.
+  - For Authy, [restore your access](https://support.authy.com/hc/en-us/articles/115012672088-Restoring-Authy-Access-on-a-New-Lost-or-Inaccessible-Phone) on a new device.
+  - For Google Authenticator, [transfer your codes](https://support.google.com/accounts/answer/1066447?hl=en&ref_topic=2954345#zippy=%2Ctransfer-google-authenticator-codes-to-new-phone) to a new device.
+
+If you lose access to your authenticator, start the account recovery process, as described in [Lost Access to Your Authenticator](/platform-deep-dive/cobalt-account/account-recovery/#lost-access-to-your-authenticator).
+
+To reset 2FA on your account:
```

Comment:
Rows 96 - 101 isn't this duplicate information as above?

Reply:
@molfinn This instruction describes resetting 2FA. The previous instruction is for enabling 2FA.